### PR TITLE
Update ex02_reqrep.nim

### PR DESCRIPTION
**Pull request by sillagiusti**
Fix: listen and connect were inverted

### CODE REVIEW

File: examples/ex02_reqrep.nim [hunk 1]
```diff
@@ -10,7 +10,7 @@ proc requester() =
     elem = i/10
 
   echo "listen"
```
Comment (xflywind) [inline examples/ex02_reqrep.nim:12]:
```suggestion
  echo "connect"
```

File: examples/ex02_reqrep.nim [hunk 2]
```diff
@@ -24,7 +24,7 @@ proc requester() =
 # An example receiving message until they are no more
 proc responder() {.gcsafe.} =
   echo "connect"
```
Comment (xflywind) [inline examples/ex02_reqrep.nim:26]:
```suggestion
  echo "listen"
```

File: examples/ex02_reqrep.nim (hunk 1 above)
Comment (Clonkk) [reply examples/ex02_reqrep.nim:12]: I agree it makes more sense to fix the inverted ``echo`` (or remove them entirely).

**Comment by Clonkk** [comment]
This PR has been adressed in #32

**Pull request closed without merging**